### PR TITLE
Update parameter descriptions

### PR DESCRIPTION
**Pull request by davidbuzinski**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 name: Setup MATLAB
 description: >-
-  Set up MATLAB on a Linux-based GitHub-hosted runner
+  Set up a specific version of MATLAB and MathWorks products
```
Comment (mw-hrastega) [inline action.yml:5]: Is it possible to remove "and MathWorks products"?

Comment (davidbuzinski) [reply action.yml]: Updated

File: action.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 name: Setup MATLAB
 description: >-
-  Set up MATLAB on a Linux-based GitHub-hosted runner
+  Set up a specific version of MATLAB
```
Comment (mw-hrastega) [inline action.yml:5]: On line 9, I see we still say R2020a or later. Shouldn't it now be R2021a or later?

Comment (davidbuzinski) [reply action.yml:5]: Good catch! updated

**Review by mw-hrastega — APPROVED** [review]

**Pull request merged**